src/test/java/org/jboss/xavier/integrations/route/ToBeanRouter_DirectToSortBean.java

```diff
@@ -0,0 +1,161 @@
+package org.jboss.xavier.integrations.route;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.test.spring.CamelSpringBootRunner;
+import org.apache.camel.test.spring.MockEndpointsAndSkip;
+import org.apache.camel.test.spring.UseAdviceWith;
+import org.jboss.xavier.Application;
+import org.jboss.xavier.integrations.route.model.SortBean;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(CamelSpringBootRunner.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@MockEndpointsAndSkip("")
+@UseAdviceWith // Disables automatic start of Camel context
+@SpringBootTest(classes = {Application.class})
+@ActiveProfiles("test")
+public class ToBeanRouter_DirectToSortBean {
```

Comment:
Please add the `Test` suffix otherwise the test won't be executed
```suggestion
public class ToBeanRouter_DirectToSortBeanTest {
```